util/conan-package/test_package/CMakeLists.txt

```diff
@@ -3,11 +3,10 @@ cmake_minimum_required(VERSION 3.0.0)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-set(CMAKE_C_FLAGS "${CONAN_C_FLAGS}")
-set(CMAKE_CXX_FLAGS "${CONAN_CXX_FLAGS}")
-set(CMAKE_SHARED_LINKER_FLAGS "${CONAN_SHARED_LINKER_FLAGS}")
+if(MSVC)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
+endif(MSVC)
 
 add_executable(testGlm main.cpp)
-TARGET_COMPILE_DEFINITIONS(testGlm PUBLIC "${CONAN_DEFINES}")
-TARGET_LINK_LIBRARIES(testGlm PUBLIC "${CONAN_LIBS}")
-SET_TARGET_PROPERTIES(testGlm PROPERTIES LINK_FLAGS "${CONAN_EXE_LINKER_FLAGS}")
+
```

Comment:
Thanks for the feedback. I've also updated the readme: https://github.com/dimi309/glm/blob/master/util/conan-package/README.md

I don't see how the conan packaging configuration could possibly affect the build. It is located within its own directory and, since it requires no compilation, it doesn't even have a travis or appveyor setup.